BootloaderCorePkg/BootloaderCorePkg.dsc

```diff
@@ -396,4 +396,9 @@
   *_*_*_CC_FLAGS = -DLITE_PRINT
 !endif
 
-
+  # -O0 results in too big size. Override it to -O1 with lto
+!if $(TARGET) == NOOPT
+  *_GCC5_IA32_CC_FLAGS = -flto -O1

```

Comment:
can we make it more generic ?

Reply:
I will do as below.
*_GCC49_*_CC_FLAGS = -O1
*_GCC49_*_DLINK_FLAGS = -O1
*_GCC5_*_CC_FLAGS = -flto -O1
*_GCC5_*_DLINK_FLAGS = -flto -O1